src/Project/ProjectService.ts

```diff
@@ -0,0 +1,25 @@
+import { INTERNAL_SERVER_ERROR, NOT_FOUND } from "../Resources/constants/statusCodes";
+import { getXataClient } from "../xata";
+import { HttpException } from "../Resources/exceptions/HttpException";
+const xata = getXataClient();
+
+class ProjectService {
+
+
+    async getProjectDetails(projectId: string, userId: string) {
+
+        const record = await xata.db.Project
+            .select(["assetController", "assetType", "enabledPaymentMethods.*", "web3Environment",
```

Comment:
Picking only what we need from the DB, that's really cool.
I noticed that "enabledPaymentMethods.*" is mentioned twice, oversight?

Reply:
thanks, mistake.